src/article.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
```

Comment:
not a huge deal but typically component file names are capitalized. like Article.js.